ui/temboardui/cli/query_agent.py

```diff
@@ -22,6 +22,12 @@ class QueryAgent(SubCommand):
     name = "query-agent"
 
     def define_arguments(self, parser):
+        parser.add_argument(
+            "--delete", action="store_const", const="DELETE", dest="method"
+        )
+
+        parser.add_argument("--post", action="store_const", const="POST", dest="method")
```

Comment:
Is there a case where we can do a POST without a `body`. If not, then, I don't see any real value adding the `--post` argument.

Reply:
Pour /pgconf/reload

Reply:
Pour /pgconf/reload